lib/minfraud/components/report/transaction.rb

```diff
@@ -73,6 +80,21 @@ def initialize(params = {})
           @notes           = params[:notes]
           @transaction_id  = params[:transaction_id]
           self.tag         = params[:tag]
+
+          validate
+        end
+
+        private
+
+        def validate
+          return if !Minfraud.enable_validation
+
+          if ip_address.nil? &&
+             minfraud_id.nil? &&
```

Comment:
Should we check if that's empty too? Possibly we could validate it further. Maybe some of the others too.